README.md

```diff
@@ -2,13 +2,13 @@
 
 [![Build Status](https://img.shields.io/travis/Turbo87/eslint-plugin-chai-expect/master.svg)](https://travis-ci.org/Turbo87/eslint-plugin-chai-expect)
 
-ESLint plugin that checks for common chai.js expect() mistakes
+ESLint plugin that checks for common chai.js `expect()` mistakes
 
 
 ## Requirements
 
 - Node.js 6 or above
-- ESLint 4.x or 5.x
+- ESLint 4.x or 5.x or 6.x
```

Comment:
we don't currently test this one on CI. the other changes are 👍 but I'd like to skip this one until we add ESLint 6 to the CI matrix.

Reply:
Ok, I've removed it here, and attached it to the CI PR which is more of an ESLint 6 upgrade PR anyways.